fixtures/uking/300w-stage-spitfire-three-headed-spitfire-pulse-ignition-three-headed-real-flame-machine-dmx-control-concert-performance-party-atmosphere.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "300W Stage Spitfire Three-headed Spitfire (Pulse Ignition) Three-headed Real Flame Machine DMX Control Concert Performance Party Atmosphere",
+  "shortName": "3-Headed Spitfire Machine",
```

Comment:
I'm not crazy about such an outrageous name (or filename), but this does appear to be the actual title that they use.  The manual says "Three head flamethrower" (and nothing else at all) on the title page, though that's more of a description than a title.

It would be nice to include the model number, "ZQ30002", somewhere, even if just in a comment.